packages/app/app.json

```diff
@@ -22,7 +31,11 @@
       "**/*"
     ],
     "ios": {
+      "splash": {
+        "image": "./assets/splash.png",
```

Comment:
please create an `images` folder in the `assets` folder, feel free to change the gitignore if you need to.
please create named folders to organize the images so we know what part of the app those images belong to.
please make the names of the files a little more descriptive so we know what those images belong to.